From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Keeley Hammond <vertedinde@electronjs.org>
Date: Tue, 1 Aug 2023 11:27:30 -0700
Subject: build: remove enterprise content analysis assert

Chromium is using enterprise content analysis, which is something
Electron is not concerned with. Remove this assert before building.
This patch can be removed when we decide how we want to handle
enterprise content analysis.

diff --git a/components/enterprise/buildflags/buildflags.gni b/components/enterprise/buildflags/buildflags.gni
index cbfe3299d44d0f34ebd1459f61707172753874f3..1ff8921df54e32396225f52d001bbf85c703a5cd 100644
--- a/components/enterprise/buildflags/buildflags.gni
+++ b/components/enterprise/buildflags/buildflags.gni
@@ -41,4 +41,4 @@ declare_args() {
 # These two flags are equivalent, but `enable_print_content_analysis` still
 # needs to exist separately as it lives under //printing which cannot depend
 # on //components.
-assert(enterprise_content_analysis == enable_print_content_analysis)
+# assert(enterprise_content_analysis == enable_print_content_analysis)
